Backend/PostgresConnector.py

```diff
@@ -5,49 +5,45 @@
 # might want to use database entry ID instead of label for identifying specific systems
 
 # gets all systems from the database
+
```

Comment:
In order for this approach to work, you'll need to turn this PostgressConnector into a class (and pass self as a parameter to all the functions of PostgressConnector).